ci_framework/roles/edpm_build_images/tasks/run.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Set DIB elements path
+  ansible.builtin.set_fact:
+    cifmw_elements: "{{ cifmw_edpm_build_images_elements | join(':') }}"
+    cacheable: true
+
+- name: Build images
+  when:
+    - (cifmw_edpm_build_images_all | bool) or (cifmw_edpm_build_images_hardened_uefi | bool) or (cifmw_edpm_build_images_ironic_python_agent | bool)
+  block:
+    - name: Build EDPM hardened uefi image
+      ansible.builtin.shell: |
+        export ELEMENTS_PATH={{ cifmw_elements }}
```

Comment:
Please refer: https://logserver.rdoproject.org/openstack-periodic-integration-stable1/opendev.org/openstack/tripleo-ci/master/periodic-tripleo-centos-9-buildimage-overcloud-hardened-uefi-full-wallaby/0e8ddf7/build_images.sh

we need support for these parameters:-

* DIB_LOCAL_IMAGE
* DIB_YUM_REPO_CONF
* DIB_DEBUG_TRACE
* DISTRO_NAME
* DIB_RELEASE
* DIB_DEBUG_TRACE

Maybe we can add a template similiar to https://github.com/openstack/tripleo-ci/blob/master/roles/oooci-build-images/templates/build-images.sh.j2 with correct tunings.

Reply:
done